libs/docs/platform/multi-combobox/examples/multi-combobox-columns/multi-combobox-columns-example.component.ts

```diff
@@ -11,15 +11,15 @@ import { MultiComboboxSelectionChangeEvent } from '@fundamental-ngx/platform/for
 })
 export class MultiComboboxColumnsExampleComponent {
     dataSource = [
```

Comment:
Yes we need to provide best practises how to work with objects, they need to provide an ID, otherwise when lookup key is not set we should throw warning message